masterlist.yaml

```diff
@@ -1113,6 +1117,14 @@ plugins:
   - name: '(Inventory )?Sorter - (Ammo Supplies|Books|Cards|Combined (- Aid Crafting|- T3T Icon Compatch|RUS_(1|2))|Consumables|Crafting Parts|Currency|Magazines|Moddable Unique Weps|Mods)\.esp'
     msg: [ *obsolete ]
 
+  - name: 'Vanilla UI Plus.esp'
+    url:
+      - link: 'https://www.moddb.com/mods/vanilla-ui-plus/downloads/vanilla-ui-plus-nv'
+        name: 'Vanilla UI Plus'
+    clean:
+      - crc: 0x57E716B8
+        util: 'FNVEdit v4.0.4c'
+
```

Comment:
Please remove this entry from this PR and create a new one for Vanilla UI Plus.

Reply:
Yeah somehow I was making a PR for vanilla UI plus and I dind't do it properly, no worries.